client/my-sites/customer-home/cards/education/educational-content/index.jsx

```diff
@@ -81,9 +84,21 @@ export default function EducationalContent( {
 			</div>
 			{ isDesktop() && (
 				<div className="educational-content__illustration">
-					<img src={ illustration } alt="" />
+					<img src={ illustration } alt="" width={ width } height={ height } />
 				</div>
 			) }
 		</div>
 	);
 }
+
+EducationalContent.propTypes = {
+	title: PropTypes.string,
+	description: PropTypes.node.isRequired,
+	links: PropTypes.array,
+	modalLinks: PropTypes.array,
+	illustration: PropTypes.string,
+	cardName: PropTypes.string,
+	width: PropTypes.oneOfType( [ PropTypes.string, PropTypes.number ] ).isRequired,
```

Comment:
I think width and height are only required if the illustration is provided. So, the following one or something like `requiredIf` might be better
```js
width: (props, propName, componentName) => {
  if ( ! props[ 'illustration' ] ) {
    return;
  }

  const prop = props[ propName ];
  if ( typeof prop !== 'string' && typeof prop !== 'number' ) {
    return new Error( `${ componentName } requires ${ propName } if illustration is provided` );
  }
},
height : (props, propName, componentName) => {
  // the same as above
},
```

		children: ( props, propName, componentName ) => {
			const prop = props[ propName ];
			return (
				Children.toArray( prop ).find( ( child ) => child.type !== TimelineEvent ) &&
				new Error( `${ componentName } only accepts "<TimelineItem />" elements` )
			);
		},

Reply:
Make sense. Thanks for this suggestion.